src/components/common/BottomsheetComponent.jsx

```diff
@@ -0,0 +1,314 @@
+import { useEffect, useRef } from 'react';
+import { createPortal } from 'react-dom';
+import styled from 'styled-components';
+import { useBottomsheetStore } from '../../stores/useBottomsheetStore';
+
+// 공통 styled-components
+const SHandleBar = styled.div`
+  display: flex;
+  position: relative;
+  width: 100%;
+  height: 1.5rem;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+  border-top-left-radius: 1rem;
+  border-top-right-radius: 1rem;
+  cursor: grab;
+  touch-action: none;
+  user-select: none;
+
+  &:active {
+    cursor: grabbing;
+  }
+
+  &::before {
+    content: '';
+    width: 2.5rem;
+    height: 0.25rem;
+    background-color: #d1d5db;
```

Comment:
❗ Chakra UI에서 정의하는 CSS 변수를 사용해 주세요.
```suggestion
    background-color: var(--colors-bg-emphasized);
```